src/index.html

```diff
@@ -10,13 +10,67 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <img
+        src="images/logo.png"
+        alt="Logo Image"
+      />
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a
+              href="apple"
+              class="is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="nav_item">
+            <a href="samsung">samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="smartphones">smartphones</a>
+          </li>
+
+          <li class="nav_item">
+            <a
+              href="Laptops & Computers"
```

Comment:
The `href` attribute value 'Laptops & Computers' contains spaces, which is not a valid URL format. Consider replacing spaces with hyphens or underscores, or encoding the spaces as '%20'.